File: C++/Medium/WrongAnswer/longest-nice-subarray_8.cpp
class Solution {
public:
    int longestNiceSubarray(vector<int>& nums) {
        
        int n = nums.size();
        vector<int> dp(n+1);
        int res = 0;
        for (int i = 0; i < n; ++i) {
            dp[i] = nums[i];
            for (int j = i-1; j >= 0; --j) {
                dp[j] &= nums[i];
                if (dp[j] == 0) {
                    res = max(res, i-j);
                }
            }
        }
        return res;
    }
};

File: Java/Medium/RuntimeError/longest-nice-subarray_8.java
class Solution {
    public int longestNiceSubarray(int[] nums) {
        
        int n = nums.length;
        int[] count = new int[26];
        int[] count1 = new int[26];
        int res = 0;
        for (int i = 0; i < n; i++) {
            Arrays.fill(count, 0);
            Arrays.fill(count1, 0);
            for (int j = i; j < n; j++) {
                if (nums[j] >= 97) {
                    count1[nums[j] - 97]++;
                } else {
                    count[nums[j] - 65]++;
                }
                if (Arrays.equals(count, count1)) {
                    res = Math.max(res, j - i + 1);
                }
            }
        }
        return res;
    }
}